releasenotes/master-remote.adoc

```diff
@@ -0,0 +1,19 @@
+:context: online
+
+= Release Notes for Open Liberty 19.0.0.10 on Red Hat OpenShift Container Platform
+
+== Features
+
+include::https://raw.githubusercontent.com/OpenLiberty/blogs/master/publish/2019-10-11-configure-logs-JSON-format-190010.adoc[leveloffset=+1,lines=18..24;26..57,63..126]
+
+== Resolved Issues
+
+See the list of resolved issues for https://github.com/OpenLiberty/open-liberty/issues?q=label%3A%22release+bug%22+label%3Arelease%3A190010+is%3Aclosed[Open Liberty 19.0.0.10] to view the list of issues that have been resolved for this release.
+
+== Fixed CVEs
+
+See the https://openliberty.io/docs/ref/general/#security-vulnerabilities.html[security vulnerability] list to view the list of CVEs fixed in Open Liberty 19.0.0.10.
+
+== Known Issues
+
+For a list of issues found during the development of 19.0.0.10 that were not fixed in the release see this https://github.com/OpenLiberty/open-liberty/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22release+bug%22+created%3A2019-09-06..2019-10-04+-label%3Arelease%3A190010+[list].
```

Comment:
@NottyCode - 2 suggestions below: 

* Change
"Known Issues" to "Known issues"

* Change
`For a list of issues found during the development of 19.0.0.10 that were not fixed in the release see this https://github.com/OpenLiberty/open-liberty/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22release+bug%22+created%3A2019-09-06..2019-10-04+-label%3Arelease%3A190010+[list].`
To
`See the https://github.com/OpenLiberty/open-liberty/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22release+bug%22+created%3A2019-09-06..2019-10-04+-label%3Arelease%3A190010+[list of issues that were found but not fixed during the development of 19.0.0.10].`